documentation/smg/11_ISIS_release.md

```diff
@@ -32,48 +48,44 @@ Code base for bugfix purposes is linked directly to the git repository for suppo
 `/usr/local/mprogs/Herbert_bugfix` -> `/home/isis_direct_soft/Herbert_bugfix`
 `/usr/local/mprogs/Horace_bugfix` -> `/home/isis_direct_soft/Horace_bugfix`
 
-#### Deploy Process
+Recent Horace version does not need separate Horace, so only second link should be used for running bugfix code.
+The code is invoked by calling `horace_on` with the path to `Horace_bugfix` folder, i.e: `horace_on("/usr/local/mprogs/Horace_bugfix")`
 
-As soon as release branch is tagged and created in the repository, manual build/release process is preformed in the private area of an ISISCOMPUTE or iDaaaS machine to obtain release artefacts for the computational service. It is also possible to pull the release artefacts from Jenkins builds, though this process needs thorough testing. 
-The extraction of the release artefacts will be automated in a future, as [iDaaaS testing pipeline](https://github.com/pace-neutrons/Horace/issues/271) is implemented. The releaser then manually checks out the release branch into `Herbert_git/Horace_git` areas above and copies the release artefacts into appropriate places of the code tree.
 
-#### Notes
+## Deploy Process
 
-The script which automates these operations and performs internal releases as one-click operation will be developed in the nearest future.
+As soon as release branch is tagged and created in the repository, manual build/release process is preformed in the private area of iDaaaS machine to obtain release artefacts for the iDAAaS computational service. It is also possible to pull the release artefacts from Jenkins builds, though this process needs thorough testing. 
+The extraction of the release artefacts will be automated in a future, as [iDaaaS testing pipeline](https://github.com/pace-neutrons/Horace/issues/271) is implemented.
 
-The members of **mslice** group on ISISCOMPUTE have write access to release code areas. Alex Buts or FBU IT support team can currently add users to **mslice** group on ISISCOMPUTE. Normal users have read-only access to the code area, sufficient to use the code in their computations.
+The releaser then manually checks out the release branch into `Horace_git` areas above and copies the release artefacts into appropriate places of the code tree.
 
-## IDAaaS
+Minor changes and bugfixes do not need the rebuild of the release artefacts and performed by pulling recent merges to the release branch into `Horace_git` repository.
```

Comment:
you don't say WHAT is performed. The verb needs a subject.

Reply:
hopefully clarified